src/sc_client/session.py

```diff
@@ -129,6 +135,9 @@ def send_message(request_type: common.ClientCommand, payload: Any) -> Response:
             common.PAYLOAD: payload,
         }
     )
+    len_data = len(bytes(data, "utf-8"))
+    if len_data > MAX_PAYLOAD_SIZE:
+        raise PayloadMaxSizeError(f"Data is too large: {len_data} > {MAX_PAYLOAD_SIZE} bytes")
```

Comment:
Can you add tests for checking error raising?

Reply:
Added